app/2.0.x/db-less-admin-api.md

```diff
@@ -14,9 +20,9 @@ service_body: |
     `connect_timeout`<br>*optional* |  The timeout in milliseconds for establishing a connection to the upstream server.  Defaults to `60000`.
     `write_timeout`<br>*optional* |  The timeout in milliseconds between two successive write operations for transmitting a request to the upstream server.  Defaults to `60000`.
     `read_timeout`<br>*optional* |  The timeout in milliseconds between two successive read operations for transmitting a request to the upstream server.  Defaults to `60000`.
-    `tags`<br>*optional* |  An optional set of strings associated with the Service, for grouping and filtering.
+    `tags`<br>*optional* |  An optional set of strings associated with the Service, for grouping and filtering. 
```

Comment:
There are several "trailing spaces" like this one which are removed. Those are fine.